src/sonic-yang-models/yang-models/sonic-events-dhcp-relay.yang

```diff
@@ -0,0 +1,73 @@
+module sonic-events-dhcp-relay {
+    namespace "http://github.com/sonic-net/sonic-events-dhcp-relay";
+    yang-version 1.1;
+    prefix events-dhcp-relay;
+
+    import sonic-events-common {
+        prefix evtcmn;
+        revision-date 2022-12-01;
+    }
+
+    import sonic-port {
+        prefix port;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC dhcp-relay events";
+
+    revision 2022-12-01 {
+        description "dhcp-relay alert events.";
+    }
+
+    container sonic-events-dhcp-relay {
+        container dhcp-relay-discard {
+            evtcmn:ALARM_SEVERITY_MAJOR;
+
+            description "
+                Declares an event for dhcp-relay discarding packet on an
+                interface due to missing IP address assigned.
+                Params:
+                    name of the interface discarding.";
+
+            leaf ifname {
+                type leafref {
+                    path "/port:sonic-port/port:PORT/port:PORT_LIST/port:name"; 
+                }
```

Comment:
Is it possible that this interface is a portchannel?

Reply:
This message is on physical i/f only (with HW address). The INTERFACE table tags ports with IP address. This INTERFACE table too refers to PORT table only.

Reply:
Added PortChannel